### PR TITLE
Prep for 0.2.0 release

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
- Update features to reflect our changes in plan
- Update telemetry notice to use generic text with link to privacy statement (as recommended)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
+## 0.2.0 - 2017-11-10
 ### Added
-DocumentDB and MongoDB accounts now support:
-- View/Create/Delete databses, collections, and documents
-- Click on a document to open in the editor
-- Edit a document and persist changes to the cloud
-- 'Load more' documents in the explorer
+- Graph:
+  - View/Create/Delete databases and graphs
+  - Click on a graph to open graph viewer
```
Comment (StephenWeatherford) [inline CHANGELOG.md]: How about 

- Click on a graph to visualize data
- Query graph using gremlin (see https://docs.microsoft.com/en-us/azure/cosmos-db/gremlin-support)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
+- DocumentDB:
+  - View/Create/Delete databses, collections, and documents
```
Comment (StephenWeatherford) [inline CHANGELOG.md]: databases

File: README.md [hunk 3]
```diff
@@ -11,16 +11,13 @@
 
 - Create a Cosmos DB account by clicking the `+` button in the title
 - View Azure Cosmos DB accounts and open directly in the portal
-- Attach a Mongo server by clicking the plug icon in the title
-  - Typical connection string for a local MongoDB instance: `mongodb://localhost:27017`
-
-MongoDB and DocumentDB accounts* support the following:
-- View/Create/Delete databses, collections, and documents
+- View/Create/Delete databses, collections, graphs, and documents
```
Comment (StephenWeatherford) [inline README.md]: ditto

File: README.md [hunk 4]
```diff
@@ -11,16 +11,13 @@
...
 - Create a Cosmos DB account by clicking the `+` button in the title
 - View Azure Cosmos DB accounts and open directly in the portal
-- Attach a Mongo server by clicking the plug icon in the title
-  - Typical connection string for a local MongoDB instance: `mongodb://localhost:27017`
-
-MongoDB and DocumentDB accounts* support the following:
-- View/Create/Delete databses, collections, and documents
+- View/Create/Delete databses, collections, graphs, and documents
 - Click on a document to open in the editor
+- Click on a graph to open graph viewer
```
Comment (StephenWeatherford) [inline README.md]: ditto

**Comment by StephenWeatherford** [comment]
We should add a bitmap of the viewer to read.me

**Comment by EricJizbaMSFT** [comment]
Let's do the gif in a separate PR - those take time to create and I'd rather do it after we have the 'final' build

**Pull request merged**